app/views/privacy_policies/show.html.erb

```diff
@@ -0,0 +1,162 @@
+<% content_for :before_content, govuk_back_link(text: "Back", href: :back) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Who we are and why we process personal data</h1>
+    <p>We are Teacher Continuing Professional Development (CPD), within the Department for Education (DfE).</p>
+    <p>The Early Career Framework (ECF) is an initiative funded by CPD.</p>
+    <p>Mentions of “us” and “we” means CPD and “you” means anyone using this service.</p>
+
+    <h2 class="govuk-heading-l">How we look after your personal data</h2>
+    <p class="govuk-body">Any information we collect when you use our services will be used and looked after by:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>the Department for Education</li>
+      <li>the teacher training providers the candidate applies to</li>
+    </ul>
+    <p class="govuk-body">
+      This means we’re joint data controllers, as well as data processors, for your information (defined by data protection law).
+    </p>
+    <p class="govuk-body">
+      The Support for Early Career Teachers Service is a new DfE service.
+      We’re confident that it has met our security standards, but we’ll continue to run security tests as we improve the service.
+    </p>
+
+    <h2 class="govuk-heading-l">Why our use of your personal data is lawful</h2>
+    <p>
+      In order for our use of your personal data to be lawful, we need to meet one, or more, conditions in the data protection legislation.
+    </p>
+    <p>
+      DfE processes your data under the basis of <strong>Public Task (Article 6(1)(e) of GDPR)</strong>.
+      In this case, it is necessary for the department to carry out this work, and to use the information for these purposes, to exercise its functions.
+    </p>
+
+    <h2 class="govuk-heading-l">What personal data we collect</h2>
+    <p class="govuk-body">
+      If you’re a candidate using the Support for Early Career Teachers Service we will collect the following information about you:
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>your preferred name</li>
+      <li>your email address</li>
+    </ul>
+
+    <h2 class="govuk-heading-l">How we use your data</h2>
+    <h3 class="govuk-heading-m">Tracking progress through self study materials</h3>
+    <p class="govuk-body">Tracking progress through self study materials includes the following:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>Identify which self study materials are relevant to you</li>
+      <li>identifying which self study materials have been accessed</li>
+      <li>getting in touch if there’s a security issue concerning your data</li>
+      <li>getting in touch with candidates and schools about progress through self study materials</li>
+      <li>getting in touch with candidates and school about self study materials</li>
+      <li>Match, link and analyse your data along with other datasets</li>
```

Comment:
Some lines start with a capital and some don't. English isn't my strongest but could be intentional?

Reply:
I copy pasted it from a content doc...
But I should probably double check this.

Reply:
I think they should be lower case judging by other places. When Mark gets back to me I can push a copy fix, but I wanna get it in before I start working on accessibility statement.